plugins/catalog-graph/src/components/EntityRelationsGraph/EntityRelationsGraph.tsx

```diff
@@ -77,6 +78,7 @@ export type EntityRelationsGraphProps = {
   mergeRelations?: boolean;
   kinds?: string[];
   relations?: string[];
+  filterFunc?: (entity: Entity) => boolean;
```

Comment:
I'm not sure that `filterFunc` is the best name. Other suggestions?

Reply:
Or maybe call it `entityFilter` as it will filter entities out of the graph